src/lib/InstantSearch.js

```diff
@@ -112,6 +168,13 @@ Usage: instantsearch({
         ...ROUTING_DEFAULT_OPTIONS,
         ...routing,
       };
+
+    if (options.createAlgoliaClient) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        'InstantSearch.js: `createAlgoliaClient` option is deprecated and will be removed in the next major version. Please use `searchClient` instead.'
```

Comment:
Can you add a link to the documentation that explains how to do this switch? Always good to have more than just an error message, but an actual actionable link

Reply:
Yes, I'm working on a guide "Preparing for v3" that I will link in this warning message.